server/src/Metrics.hs

```diff
@@ -61,6 +62,8 @@ createAndRegisterIcepeakMetrics = IcepeakMetrics
                               "Total number of items added to the WebSocket queue."))
   <*> register (counter (Info "icepeak_internal_ws_queue_items_removed"
                               "Total number of items removed from the WebSocket queue."))
+  <*> register (counter (Info "icepeak_internal_ws_queue_skipped_updates"
+                              "Total number of updates discarded from the WebSocket queue."))
   <*> register (counter (Info "icepeak_internal_ws_skipped_updates_total"
                               "Total number of updates that have not been sent to subscribers."))
```

Comment:
Ah, I see your commit message mentions "rename old metric". Given that prometheus doesn't support renames, those are always pretty disruptive since they break existing dashboards/severely complicate their queries (until the old data is completely flushed out).

If possible, it's better to find a new descriptive name for the new metric, and maybe adjust the help texts to help disambiguate the two.

Reply:
That is right, but to be honest, I was thinking about flushing the old metric. I can try and find better less disruptive names, though.